src/main/scala/org/zalando/nakadi/client/KlientBuilder.scala

```diff
@@ -45,20 +50,22 @@ class KlientBuilder(val endpoint: URI = null, val port: Int, val securedConnecti
 
 
   def defaultObjectMapper: ObjectMapper = {
-    val mapper = new ObjectMapper
-    mapper.registerModule(new DefaultScalaModule)
-    mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
-    mapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
-    mapper.setPropertyNamingStrategy(PropertyNamingStrategy.CAMEL_CASE_TO_LOWER_CASE_WITH_UNDERSCORES)
-    mapper.addHandler(new DeserializationProblemHandler() {
-      override def handleUnknownProperty(ctxt: DeserializationContext, jp: JsonParser, deserializer: JsonDeserializer[_], beanOrClass: AnyRef, propertyName: String): Boolean = {
-        logger.warn(s"unknown property occurred in JSON representation: [beanOrClass=$beanOrClass, property=$propertyName]")
-       true
-      }
-    })
-    mapper
+    new ObjectMapper()
+      .registerModule(new DefaultScalaModule)
+      .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+      .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+      .setPropertyNamingStrategy(PropertyNamingStrategy.CAMEL_CASE_TO_LOWER_CASE_WITH_UNDERSCORES)
+      .addHandler(new DeserializationProblemHandler() {
+        override def handleUnknownProperty(ctxt: DeserializationContext, jp: JsonParser, deserializer: JsonDeserializer[_], beanOrClass: AnyRef, propertyName: String): Boolean = {
+          logger.warn(s"unknown property occurred in JSON representation: [beanOrClass=$beanOrClass, property=$propertyName]")
+          true
+        }
+      })
```

Comment:
Matter of taste if an interim variable should be used (e.g. to help debugging). By using `()` after the construction, Scala seems to be fine with chaining the method calls.